pages/merchant/Merchant.qml

```diff
@@ -0,0 +1,697 @@
+                verticalOffset: 3
+                radius: 8.0
+                samples: 16
+                color: "#20000000"
+                smooth: true
+                source: tracker
+            }
+
+            Rectangle {
+                id: qrImg
+                color: "white"
+
+                anchors.right: parent.right
+                anchors.top: parent.top
+
+                height: root.qrCodeSize
+                width: root.qrCodeSize
+
+                Layout.maximumWidth: root.qrCodeSize
+                Layout.preferredHeight: width
+                radius: 5
+
+                Image {
+                    id: qrCode
+                    anchors.fill: parent
+                    anchors.margins: 1 * scaleRatio
+
+                    smooth: false
+                    fillMode: Image.PreserveAspectFit
+                    source: "image://qrcode/" + TxUtils.makeQRCodeString(appWindow.current_address, amountToReceive.text)
+
+                    MouseArea {
+                        anchors.fill: parent
+                        acceptedButtons: Qt.RightButton
+                        onClicked: {
+                            if (mouse.button == Qt.RightButton){
+                                qrMenu.x = this.mouseX;
+                                qrMenu.y = this.mouseY;
+                                qrMenu.open()
+                            }
+                        }
+                        onPressAndHold: qrFileDialog.open()
+                    }
+                }
+
+                Menu {
+                    id: qrMenu
+                    title: "QrCode"
+
+                    MenuItem {
+                        text: qsTr("Save As") + translationManager.emptyString;
+                        onTriggered: qrFileDialog.open()
+                    }
+                }
+            }
+
+            DropShadow {
+                anchors.fill: source
+                cached: true
+                horizontalOffset: 3
+                verticalOffset: 3
+                radius: 8.0
+                samples: 16
+                color: "#30000000"
+                smooth: true
+                source: qrImg
+            }
+        }
+
+        Item {
+            Layout.preferredHeight: 40 * scaleRatio
+            anchors.left: parent.left
+            anchors.right: parent.right
+
+            Item {
+                width: (parent.width - qrImg.width) - (50 * scaleRatio)
+                height: 32 * scaleRatio
+
+                Text {
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    font.pixelSize: 12 * scaleRatio
+                    font.bold: false
+                    color: "white"
+                    text: "<style type='text/css'>a {text-decoration: none; color: #FF6C3C; font-size: 12px;}</style>Currently selected address: " + addressLabel + " <a href='#'>(Change)</a>"
```

Comment:
qsTr and place the tags outside

Reply:
See previous reply regarding mixing of tags/translatable strings.

Reply:
:ok_hand: